.cypress/integration/2_notebooks.spec.js

```diff
@@ -41,28 +50,30 @@ describe('Adding sample data and visualization', () => {
   });
 
   it('Add sample observability data', () => {
-    moveToPanelHome();
-    cy.get('.euiButton__text').contains('Actions').trigger('mouseover').click();
+    moveToEventsHome();
+    cy.get('button[data-test-subj="eventHomeAction"]').trigger('mouseover').click();
     cy.wait(100);
-    cy.get('.euiContextMenuItem__text').contains('Add samples').trigger('mouseover').click();
+    cy.get('button[data-test-subj="eventHomeAction__addSamples"]').trigger('mouseover').click();
     cy.wait(100 * 3);
     cy.get('.euiModalHeader__title[data-test-subj="confirmModalTitleText"]')
       .contains('Add samples')
       .should('exist');
     cy.wait(100);
-    cy.get('.euiButton__text').contains('Yes').trigger('mouseover').click();
+    cy.get('button[data-test-subj="confirmModalConfirmButton"]').trigger('mouseover').click();
     cy.wait(100 * 5);
     cy.route2('POST', '/addSamplePanels').as('addSamples');
     cy.wait('@addSamples').then(() => {
-      cy.get('.euiTableCellContent').contains(SAMPLE_PANEL).should('exist');
+      // cy.get('.euiTableCellContent').contains(SAMPLE_PANEL).should('exist');
```

Comment:
can you remove the commented lines?

Reply:
thanks for the catch removed the comment.